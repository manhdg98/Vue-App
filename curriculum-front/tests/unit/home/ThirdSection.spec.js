import { mount } from '@vue/test-utils'
import ThirdSection from '@/components/home/ThirdSection.vue'

describe('ThirdSection.vue', () => {
  test('if snapshot is correct', () => {
    const wrapper = mount(ThirdSection)
    expect(wrapper).toMatchSnapshot()
  })
})
