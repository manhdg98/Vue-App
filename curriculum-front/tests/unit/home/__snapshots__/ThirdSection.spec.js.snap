// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThirdSection.vue if snapshot is correct 1`] = `
<div class="row home-section-three align-center">
  <div class="col-sm-4 offset-sm-2 col">
    <h2>Achieve Your Goals</h2>
    <p>
      Check off tasks and projects as you go and receive
      badges as you achieve your goals.
    </p>
  </div>
  <div class="col-sm-4 col"><img src="img/display_curriculum.png" class="demo-screenshot"></div>
</div>
`;
